docs/CONTRIBUTING.md

```diff
@@ -357,14 +380,51 @@ You can run individual unit tests by using the following syntax:
 Or by adding `.only` after a `describe` or `it` block in the test file:
 
 ```typescript
-describe.only('Extension manifest tests', () => {
-  it('should register all commands', () => {
+describe.only("Extension manifest tests", () => {
+  it("should register all commands", () => {
     // test code
   });
 });
 ```
 
-Remember to remove `.only` after running the test and before making a PR! 
+Remember to remove `.only` after running the test and before making a PR!
+
+#### Clicktesting the VS Code extension against the sidecar
+
+##### Method 1:
+
+Manually trigger the "Multi Arch Builds and Upload to GitHub Releases" promotion in the sidecar
+repo. This can be found in the Semaphore UI under the "Promotions" tab. This won't actually upload
+to a release, but will make the native executables available at the workflow's artifacts/ part of
+the Semaphore UI. Then you can continue on from step 3 of Method 2.
+
+##### Method 2:
+
+1. Checkout branch you want to test against in the sidecar.
+
+2. Run `make clean mvn-package-native-no-tests` in the sidecar to get the native executable packaged
+   up in the target directory.
+
+3. Copy the native executable from the sidecar's `target` directory to the `vscode/bin/` directory
+   (you can use click & drag).
```

Comment:
Nit: not really necessary to mention clicking and dragging
```suggestion
3. Copy the native executable (`ide-sidecar-{version}-runner`) from the sidecar's `target` directory into the `vscode/bin/` directory.
```

Reply:
Assuming that `$VS_CODE_EXTENSION_PROJECT` holds the path to the VS Code extension on the user's machine, they could run the following command in the root directory of the ide-sidecar project to copy the native executable:

`cp ./target/ide-sidecar-0.*.0-runner $VS_CODE_EXTENSION_PROJECT/bin`